src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure implements AreaCalculetion {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class declaration.